### ISSUE TITLE
Bug: Notebooks Error Thresholds Unusable

### ISSUE THREAD

**Issue by jzahnd** [influxdata/ui#3797]
Currently, configuring Error Thresholds in Notebooks is not usable due to space limitations. We would like to update the design to better use the space provided.

Before:
<img width="431" alt="Error Thresholds - Before" src="https://user-images.githubusercontent.com/81173353/152851527-1f1b0375-fbb9-412d-98a7-2efa6e4cad92.png">

After:
<img width="398" alt="Error Thresholds - After" src="https://user-images.githubusercontent.com/81173353/152851548-0a95000b-94e8-410d-9b75-df29fd58b8c6.png">

[Figma](https://www.figma.com/file/W49dw2Xls8FYycWEYMnkze/Customize-Visualization?node-id=18%3A23)

**Comment by asalem1** [comment]
@juliajames12 can we get some designs on how this looks in an empty state. Also, is there a design for how this looks when a user wants to delete the threshold?

**Comment by jzahnd** [comment]
Good catch Ari. I am adding needs design and moving to blocked for the very moment
